e2e/testcases/acme_test.go

```diff
@@ -190,6 +191,17 @@ func TestAcmeCorpRepo(t *testing.T) {
 	nt.WaitForRepoSyncs()
 }
 
+func TestObjectInCMSNamespace(t *testing.T) {
+	nt := nomostest.New(t, ntopts.SkipMonoRepo, ntopts.Unstructured)
+
+	nt.RootRepos[configsync.RootSyncName].Copy("../../examples/object-in-cms-namespace", "acme")
+	nt.RootRepos[configsync.RootSyncName].CommitAndPush("adding resource to config-management-system namespace")
+	nt.WaitForRepoSyncs()
+	namespace := "config-management-system"
```

Comment:
Use `configmanagement.ControllerNamespace` from the `pkg/api/configmangement` pkg instead.

Reply:
changed to use ` configmanagement.ControllerNamespace `